lib/logstash/filters/jdbc/read_write_database.rb

```diff
@@ -31,6 +31,9 @@ def build_db_object(db_object)
       begin
         @rwlock.writeLock().lock()
         db_object.build(@db)
+        if db_object.index_columns.empty?
+          logger.warn("db_object '#{db_object.name}': `index_columns` is optional but on larger datasets consider adding an index on the lookup column, it will improve performance")
```

Comment:
nit: remove `db_object` from message. 

nit: a warning should be a bit less friendly :)  suggested:  '#{db_object.name}': `index_columns` has not been defined and may cause performance issues on larger datasets.

Reply:
Agree with less friendly message.

Well, looking at the final line logged, without `do_object` or perhaps `local_db_object`, is it a bit sparse and unhelpful.
```
[2018-05-14T17:04:18,842][WARN ][logstash.filters.jdbc.readwritedatabase] 'servers': `index_columns` has not been defined and may cause performance issues on larger datasets.
```
vs
```
[2018-05-14T17:04:18,842][WARN ][logstash.filters.jdbc.readwritedatabase] local_db_object 'servers': `index_columns` has not been defined and may cause performance issues on larger datasets.
```